### PR TITLE
Update pl.ts

### PR DESCRIPTION
**Pull request by atlochowski**

### CODE REVIEW

File: translations/pl.ts [hunk 1]
```diff
@@ -247,17 +247,17 @@
...
     <message>
         <source>Elevation</source>
-        <translation type="unfinished"></translation>
+        <translation>Przewyższenie</translation>
     </message>
     <message>
         <source>%1 m a.s.l.</source>
         <extracomment>elevation - meters above sea level</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Przewyższenie - m.n.p.m.</translation>
```
Comment (Karry) [inline translations/pl.ts]: It seems to be incorrect, `%1` placeholder for value is missing...

File: translations/pl.ts [hunk 2]
```diff
@@ -247,17 +247,17 @@
...
     </message>
     <message>
         <source>Created</source>
         <extracomment>date time when waypoint was created</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>data utworzenia punktu drogi</translation>
```
Comment (Karry) [inline translations/pl.ts]: `extracomment` is just comment for translators, you should translate `source`... :-)

Comment (atlochowski) [reply translations/pl.ts]: sorry, my mistake

Comment (Karry) [reply translations/pl.ts]: Would not be just "Utworzenia" better?

Snimek_20190805_001

Comment (atlochowski) [reply translations/pl.ts]: I again translated extracomment. I will review my last commit today.

**Comment by atlochowski** [comment]
done. sorry for lazyness.

**Comment by Karry** [comment]
No problem, there is no reason to hurry, I will release next version in September, after my vacation... Thank you for translation :-)

**Pull request merged**